proto/sge/reward/campaign.proto

```diff
@@ -18,45 +18,52 @@ message Campaign {
     json_name = "uid"
   ];
 
+  // house_uid is the unique identifier of the promoter house.
+  string house_uid = 3 [
```

Comment:
it is not required. Promoter which is added below solves the problem. House id is not required.

Reply:
removed, temporarily